cmd/serve/grant.go

```diff
@@ -213,8 +230,12 @@ func setupRouter(ctx context.Context, logger *zerolog.Logger) (context.Context,
 
 	r.Mount("/v1/credentials", payment.CredentialRouter(paymentService))
 	r.Mount("/v1/orders", payment.Router(paymentService))
+	// for payment webhook integrations
+	r.Mount("/v1/webhooks", payment.WebhookRouter(paymentService))
```

Comment:
I'm not too familiar with how the routing works, is this route `/v1/webhooks` or is there like `skus/v1/webhooks`?

Reply:
will just be `/v1/webhooks` on the grant service for the time being.